### ISSUE TITLE
New table to show validators status and data

### ISSUE THREAD

**Issue by gordsport** [vegaprotocol/frontend-monorepo#799]
Needs:
- https://github.com/vegaprotocol/data-node/issues/509 
(should be done before this is started)

![image](https://user-images.githubusercontent.com/83510148/179517050-0eae72c4-a949-499d-8bf0-d2c85b968818.png)

- [ ] Remove existing list of validators (note: the new table does not include your stake, this is intentional, its in the right hand bar)
- [ ] Table is sortable by users by clicking header
- [ ] Default sort == ranking score (high to low). use ag grid?
- [ ] Table creates horizontal scroll if/when needed
- [ ] Add h2 for "Nodes"
- [ ] Consider adding a checkbox to only show validating + Stand by validators (remove ones on the longer waiting list)

The intention of this table to list all the nodes who are known / active to the network, their overall status (validating etc) and the information needed to make informed decisions about who to delegate.

There is no single key value to show someone to help inform the staking decision, therefore lots are shown. ranking score best explains what is currently happening so has been chosen as default.

This is likely to be wider than the available area, this table layout is deemed a better default than a mobile first option.

![image](https://user-images.githubusercontent.com/83510148/179517194-3ea5d727-ba11-4258-99ac-a335933b6a73.png)

- Validator (avatar and name)
- Status (Shows either Consensus, Standby* or Candidate) See https://github.com/vegaprotocol/data-node/issues/509 not to be confuses with node.status
- Total stake this epoch { nodes{ stakedTotal}}
- Share (use the total staked that epoch and the amount staked to show a share of the stake they have nominated (current epoch)
- Next epoch { nodes { pendingStake } }
- Ranking score See below
- Staked score
- Performance
- Voting power
*this may be called ersatz in the API

![image](https://user-images.githubusercontent.com/83510148/179517374-1f020f83-8630-494c-af88-a5928a19f950.png)

Future iterations may:

- an indication return per delegated VEGA https://github.com/vegaprotocol/token-frontend/issues/1277
- Highlighting issues that prevent candidate becoming an ersatz/Standby
- Show what version each node is running and what it intends to upgrade to
- Refine what columns are shown
- Consider alternatives to showing scores as floats
Make it easier to see what scores are good/bad/or neither
Consider whether to incudes normalised score

**Comment by gordsport** [comment]
from: 
- https://github.com/vegaprotocol/token-frontend/issues/1273

**Comment by gordsport** [comment]
closing as a duplicate of:
- https://github.com/vegaprotocol/frontend-monorepo/issues/45
